configs/milvus.yaml

```diff
@@ -358,6 +358,7 @@ queryNode:
       maxPendingTaskPerUser: 1024 # 50 by default, max pending task in scheduler per user.
   mmap:
     mmapEnabled: false # enable mmap global, if set true, will use mmap to load segment data
+  invertedIndexLoadPredictMemoryUsageFactor: 0.2
```

Comment:
why the memory consumption is much lesser than disk?

is this related to dataset we evaluate? since index itself has strong compression ratio? This might not be really safe? we should we directly store how much memory const for index?

Reply:
No, just because all the index will be mmapped, the actual memory occupied by the inverted index is up to the OS.

Reply:
No, just because all the index will be mmapped, the actual memory occupied by the inverted index is up to the OS.